udata_front/theme/gouvfr/templates/header.html

```diff
@@ -13,110 +15,164 @@
-                        {% if current_user.is_authenticated %}
-                        <div class="row-inline align-items-center">
-                            <img
-                                src="{{ current_user|avatar_url(24) }}"
-                                width="24"
-                                height="24"
-                                loading="lazy"
-                                alt=""
-                                class="mr-xs rounded-circle"
-                            />
-                            <span>{{ current_user.fullname }} </span>
+                </div>
+                <div class="fr-header__tools">
+                    <div class="fr-header__tools-links">
+                        <ul class="fr-links-group">
+                            {% if current_user.is_authenticated %}
+                                <li>
+                                    <div class="fr-link fr-grid-row text-grey-500">
+                                        <img
+                                            src="{{ current_user|avatar_url(24) }}"
+                                            width="24"
+                                            height="24"
+                                            loading="lazy"
+                                            alt=""
+                                            class="fr-mr-1w rounded-circle"
+                                        />
+                                        {{ current_user.fullname }}
+                                    </div>
+                                </li>
+                                <li>
+                                    <a
+                                        href="{{ url_for('admin.index') }}"
+                                        class="fr-link"
+                                    >
+                                        <span class="fr-mr-1w fr-grid-row">
+                                            {% include theme('svg/administration.svg') %}
+                                        </span>
+                                        {{ _('Administration') }}
+                                    </a>
+                                </li>
+                                <li>
+                                    <a
+                                        href="{{ url_for('security.logout') }}"
+                                        class="fr-link fr-fi-logout-box-r-line"
+                                    >
+                                        {{ _('Logout') }}
+                                    </a>
+                                </li>
+                            {% else %}
+                                <a
+                                    href="{{ url_for('security.login', next=next_url) }}"
+                                    title="{{ _('Log in') }}"
+                                    class="fr-link fr-fi-lock-line"
```

Comment:
Icons are underlined when hovered

Reply:
I removed the underline hover for DSFR link because they already have an hover state. This match what we did with buttons.